ml4h/test_evaluate.py

```diff
@@ -0,0 +1,55 @@
+import tensorflow as tf
```

Comment:
Can you do this via pytest in tests dir?

Reply:
Ah yes, we can delete this file. This was my local debugging file